packages/api/tests/endpoints/test-reconciliation-reports.js

```diff
@@ -423,25 +394,8 @@ test.serial('POST returns a 500 when failing to create an async operation', asyn
   }
 });
 
-test.serial('POST returns a 503 when there is an ecs error', async (t) => {
```

Comment:
Do we still need some way to handle providing this feedback to users?  Else it seems like in these failure cases the lambda will execute and *nothing* will show up in the DB *or* the API response.     

 In theory we could be updating the record to fail with information if `startAsyncOperation` throws, at the cost of adding database configuration to the invocation lambda.

Reply:
Updated startAsyncOperation  method to create asyncOperation record when startEcsTask fails
[fde322f](https://github.com/nasa/cumulus/pull/2962/commits/fde322ff66076714738950457eb5e7986a78c067)

Reply:
Tested manually by updating launchType to 'FARGATE2' in the code, record is saved when startEcsTask fails.
```
{
 "id": "cd27a334-5451-477d-88d6-c35c1b1e24cc",
 "createdAt": 1653505934999,
 "description": "Bulk granule reingest run on 2 granules",
 "operationType": "Bulk Granule Reingest",
 "output": "{\"name\":\"InvalidParameterException\",\"message\":\"launch type should be one of [EC2,FARGATE,EXTERNAL]\",\"stack\":\"InvalidParameterException: launch type should be one of [EC2,FARGATE,EXTERNAL]\\n    at Request.extractError (/var/runtime/node_modules/aws-sdk/lib/protocol/json.js:52:27)\\n    at Request.callListeners (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:106:20)\\n    at Request.emit (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:78:10)\\n    at Request.emit (/var/runtime/node_modules/aws-sdk/lib/request.js:686:14)\\n    at Request.transition (/var/runtime/node_modules/aws-sdk/lib/request.js:22:10)\\n    at AcceptorStateMachine.runTo (/var/runtime/node_modules/aws-sdk/lib/state_machine.js:14:12)\\n    at /var/runtime/node_modules/aws-sdk/lib/state_machine.js:26:10\\n    at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:38:9)\\n    at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:688:12)\\n    at Request.callListeners (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:116:18)\"}",
 "status": "RUNNER_FAILED",
 "updatedAt": 1653505934999
}
```

Reply:
Awesome!  Thanks for making that update!!